boa/vm/fork.py

```diff
@@ -120,7 +120,10 @@ def fetch_multi(self, payload):
             # fetch_multi is called only with the missing payloads
             # map the results back to the original indices
             for result_ix, rpc_result in enumerate(self._rpc.fetch_multi(batch)):
-                key, item_ix = keys[result_ix]
+                try:
+                    key, item_ix = keys[result_ix]
+                except IndexError:
```

Comment:
looks spooky

Reply:
sorry this shouldn't have been committed